### PR TITLE
feat:add respponsiveness of about us page

### PR DESCRIPTION
**Pull request by Mitiksha-28**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </head>
 
   <body>
-    <div id="root"></div>
+    <div id="root" style="min-width: 100vw;"></div>
```
Comment (Miran-Firdausi) [inline index.html]: `style="min-width: 100vw;`

This is causing horizontal scrolling on all pages.

File: src/pages/About/index.jsx [hunk 2]
```diff
@@ -1,129 +1,109 @@
...
-        <div className="flex flex-grow flex-wrap items-center overflow-hidden">
-          <div className="container space-y-8">
-            <p className="p-2 px-64 mx-auto m-w-12 text-xl text-wrap text-text-light text-center mt-6 space-y-16">
-              Praesent vestibulum libero non diam consequat euismod. Maecenas
-              purus sapien, mollis ut dapibus et, varius in ipsum. Morbi mollis
-              varius tincidunt. Lorem ipsum dolor sit amet, consectetur
-              adipiscing elit.
+      <div className="flex flex-col min-h-screen">
+        {/* About Us Section */}
+        <section className="bg-secondary-dark text-text-light py-12 px-4 md:px-8">
```
Comment (Miran-Firdausi) [inline src/pages/About/index.jsx]: You can use py-8 as we are using the same for others.

**Review by Miran-Firdausi — CHANGES_REQUESTED** [review]

File: src/components/Footer/index.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import { useState, useContext } from "react";
```
Comment (Miran-Firdausi) [inline src/components/Footer/index.jsx:1]: This PR should not involve Footer, right?

Comment (Mitiksha-28) [reply src/components/Footer/index.jsx:1]: yess

**Pull request merged**